tests/unit-core/test-to-property-descriptor.c

```diff
@@ -71,6 +72,13 @@ main (void)
   jerry_release_value (prop_name);
   jerry_release_value (value);
   jerry_free_property_descriptor_fields (&prop_desc);
+
+  object = jerry_create_null ();
+  result = jerry_to_property_descriptor (object, &prop_desc);
```

Comment:
What will happen if the input "object" is not an object? Ex: a simple number?

Reply:
The called function (`ecma_op_to_property_descriptor`) throws a type error. There is a test for this.